src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
+import { Button } from "react-bootstrap";
+
+
+const Home = props => {
```

Comment:
We are not using `props` so we can remove this

Reply:
Sure, I will remove it.